recipe/meta.yaml

```
@@ -8,8 +8,6 @@ package:
 source:
   url: https://github.com/rerun-io/rerun/archive/refs/tags/{{ version }}.tar.gz
   sha256: d2d3ff251a695e9e70ac3d1789afc7374fd44964a50b3780cb7ad18f62065a21
-  patches:
-    - patches/0001-Disable-building-docs-snippets.patch
 
 build:
   number: 0
```

Maybe increase the build number?

```suggestion
  number: 1
```